frontend/src/app/pages/exercises/exercise/shared/alarm-group-overview/alarm-group-overview-modal/alarm-group-overview-modal.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component } from '@angular/core';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+import { Store } from '@ngrx/store';
+import { AlarmGroup } from 'digital-fuesim-manv-shared';
+import { ApiService } from 'src/app/core/api.service';
+import type { AppState } from 'src/app/state/app.state';
+import {
+    selectAlarmGroups,
+    selectVehicleTemplates,
+} from 'src/app/state/exercise/exercise.selectors';
+
+@Component({
+    selector: 'app-alarm-group-overview-modal',
+    templateUrl: './alarm-group-overview-modal.component.html',
+    styleUrls: ['./alarm-group-overview-modal.component.scss'],
+})
+export class AlarmGroupOverviewModalComponent {
+    public exerciseId!: string;
+
+    constructor(
+        public activeModal: NgbActiveModal,
+        private readonly apiService: ApiService,
+        private readonly store: Store<AppState>
+    ) {}
+
+    public readonly alarmGroups$ = this.store.select(selectAlarmGroups);
+
+    public readonly vehicleTemplates$ = this.store.select(
+        selectVehicleTemplates
+    );
+
+    public close() {
+        this.activeModal.close();
+    }
+
+    public addAlarmGroup() {
+        this.apiService.proposeAction({
+            type: '[AlarmGroup] Add AlarmGroup',
+            alarmGroup: AlarmGroup.create('MANV 10'),
```

Comment:
Why `MANV 10`?

Reply:
As a Placeholder since groups need to have names when they are created. Maybe you could have a text input next to the add button, but I don't know if I like that. Otherwise we could use a different placeholder

Reply:
Ok, maybe use `???` or something like this instead?